### PR TITLE
AZP: Do not fail on io_demo counter

### PR DESCRIPTION
**Pull request by Alexey-Rivkin**
## What
Temporary disable job failures on io_demo counter.

### CODE REVIEW

File: buildlib/pr/io_demo/az-stage-io-demo.yaml [hunk 1]
```diff
@@ -123,7 +123,6 @@ steps:
           echo "local_ack_timeout_err counter was increased during iodemo with interference"
         else
           azure_log_error "local_ack_timeout_err counter was not increased ($(local_ack_t_err_counter_start))"
-          exit 1
```
Comment (yosefe) [inline buildlib/pr/io_demo/az-stage-io-demo.yaml:126]: shall we use azure_complete_with_issues?

**Review by yosefe — APPROVED** [review]
@Alexey-Rivkin pls squash

File: buildlib/pr/io_demo/az-stage-io-demo.yaml (hunk 1 above)
Comment (Alexey-Rivkin) [reply buildlib/pr/io_demo/az-stage-io-demo.yaml:126]: Done, 10x

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@Alexey-Rivkin can you pls port this PR to v1.15.x?

**Pull request merged**